roles/ceilometer-data/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: install ceilometer compute service
+  upstart_service: name=ceilometer-agent-compute
+                   user=ceilometer
+                   cmd=/usr/local/bin/ceilometer-agent-compute
+  notify:
+    - restart ceilometer services
+
+- name: pip install libvirt-python in venv on compute
+  pip: name=libvirt-python virtualenv={{ ceilometer.virtualenv }}
+  notify:
+    - restart ceilometer services
+  when: openstack_install_method == 'package'
```

Comment:
When installing from source nova, neutron, cinder and other projects (except for horizon) does not install in a virtualenv. We were following nova-common/meta/main.yml